compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/Conversion/ConvertCUDNNToRuntime.cpp

```diff
@@ -663,6 +683,39 @@ struct ConvertCudnnBiasOp : public CudnnOpConversionPattern<BiasOp> {
   }
 };
 
+//===----------------------------------------------------------------------===//
+// cudnn.relu
+//===----------------------------------------------------------------------===//
+
+struct ConvertCudnnReluOp : public CudnnOpConversionPattern<ReluOp> {
+  using CudnnOpConversionPattern::CudnnOpConversionPattern;
+
+  LogicalResult matchAndRewrite(
+      ReluOp op, OpAdaptor adaptor,
+      ConversionPatternRewriter &rewriter) const override {
+    MLIRContext *ctx = rewriter.getContext();
+    ImplicitLocOpBuilder b(op->getLoc(), rewriter);
+    SmallVector<Value> args(adaptor.getOperands());
+    auto f32 = rewriter.getF32Type();
+
+    // It is unable to convert the lowerClip directly to float, thus the static
+    // cast.
+    float lower_clip =
+        static_cast<float>(adaptor.getLowerClip().convertToDouble());
```

Comment:
Is `convertToFloat` an option?

Reply:
When I replaced `convertToDouble()` with `convertToFloat()` I was getting the following error:

```
iree-opt: /usr/local/google/home/bviyer/LLVM/My_Fork/iree/third_party/llvm-project/llvm/lib/Support/APFloat.cpp:5233: float llvm::APFloat::convertToFloat() const: Assertion `getSemantics().isRepresentableBy(semIEEEsingle) && "Float semantics is not representable by IEEEsingle"' failed.
```

Reply:
Ah, okay.